reactDashboard/src/components/HeatMap/Heatmap.jsx

```diff
@@ -273,37 +277,30 @@ function Heatmap({
       : "HEALTH BOARDS";
   }
 
-  function areaSubTitle() {
-    const dataset = getDataSet();
-    return (
-      dataset.regions.length +
-      " " +
-      (AREATYPE_COUNCIL_AREAS === areaType ? "Areas" : "Boards")
-    );
-  }
-
   function valueTitle() {
     return VALUETYPE_DEATHS === valueType ? "TOTAL DEATHS" : "TOTAL CASES";
   }
 
   function heatbarScale() {
     return (
-      <div className="heatmapScale">
+      <Container fluid className="heatmapScale">
```

Comment:
A container containing a single row containing a single column... why ?

Reply:
I get that the intention is to stretch the legend across the available column width, but there are cleaner ways to do that just with divs/spans. Also, could you center the text in each legend item please?

Reply:
text is centered - will come back to the other comment

Reply:
refactored